.gitignore

```diff
@@ -1,3 +1,6 @@
-.idea
 node_modules
+yarn-error.log
```

Comment:
Get rid of these changes

Reply:
Should I need to remove only `yarn-error.log` or
`yarn-lock.log`
`package-lock.json`
`yarn.lock`

Reply:
Removed `yarn-lock.log` and `package-lock.json` from `.gitignore` file.